jupyter_server/extension/manager.py

```diff
@@ -290,6 +294,13 @@ def sorted_extensions(self):
         """
     )
 
+    extension_apps = Dict(
+        help="""
+        Dictionary with extension names as keys
+        and sets of ExtensionApp objects as values.
+        """
+    )
+
```

Comment:
Let’s not make this a trait. Instead, let’s make this a property that cycles through the `self.extension_points` and returns points whose `app` attribute is not `None`.